adapters/blue/blue_test.go

```diff
@@ -0,0 +1,20 @@
+package blue
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+func TestJsonSamples(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderInMobi, config.Adapter{
```

Comment:
Please change the bidder name to `BidderBlue`.

Reply:
Fixed